spec/spec_helper.rb

```diff
@@ -27,6 +27,7 @@
   add_filter 'lib/traject/tei_config.rb'
```

Comment:
I think if we re-add `SimpleCov.start('rails') do` above, we'll get these normal filters for free, no?